tools/generator/src/DTO/Platform.swift

```diff
@@ -69,6 +93,13 @@ extension Platform.Variant {
     }
 }
 
+extension Platform.OSVersion: Comparable {
+    static func < (lhs: Platform.OSVersion, rhs: Platform.OSVersion) -> Bool {
+        return lhs.rawValue.compare(rhs.rawValue, options: .numeric)
+            == .orderedAscending
```

Comment:
We could store the components, and do a compare on each... but I'll leave that for a future optimization.